### ISSUE TITLE
Trouble with palette in R&D?

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#562]
I downloaded the game, compiled, and it's showing strange hardware type images, especially Unmanned.  Is this a problem with the color palette?

![Looks funny](https://user-images.githubusercontent.com/3118014/126883426-400c961b-ddf2-4e5f-a67a-f62adbf28bce.png)
![Looks funny 2](https://user-images.githubusercontent.com/3118014/126909070-cf7d5658-d3a4-4d28-a93b-9bd63ea776b4.png)

**Comment by rnyoakum** [comment]
Yeah, my guess is that fixing #513 overwrote some of the Soviet palette used by the hardware buttons. In that issue, the rdfull.but.*.png images weren't exporting their palettes to the display, and the fix corrected that. My guess is that the hardware button images rely upon that same palette space. So there's probably a palette conflict that will have to be resolved.

**Comment by peyre** [comment]
Ok, I'm playing again.  It all looked normal until just now when I prototyped my boosters.  So something must be happening to change the palette, or something.

![Booster](https://user-images.githubusercontent.com/3118014/127752203-225138d6-dd3b-44e1-89e8-e98d950633cf.png)

**Comment by rnyoakum** [comment]
> Ok, I'm playing again. It all looked normal until just now when I prototyped my boosters. So something must be happening to change the palette, or something.

That makes sense, per my earlier comment. The pre-prototype image isn't overwriting the palette space. Purchasing the booster stage loads and draws the rdfull.but image,  which writes over the palette space.

**Comment by peyre** [comment]
Oh!  Yes.  Well, I'm glad to help pinpoint the cause.

**Comment by peyre** [comment]
Interesting - today I noticed it doing something very similar on a turn when I'm pretty sure I hadn't prototyped anything.

![Didn't prototype anything](https://user-images.githubusercontent.com/3118014/132266372-bd7f3748-f45b-4ebe-90af-5dfa3c1d3d18.png)

**Comment by peyre** [comment]
Ok, that's confirmed.  Sometimes I get this while flipping through R&D when I haven't even been in Purchasing that turn, so couldn't have prototyped anything that turn.

**Comment by peyre** [comment]
Well I hadn't, though I had made the discoloration appear less often.  It should be fixed now.

**Comment by peyre** [comment]
Nope.  It's better, but still not quite right.  Ryan, do you think maybe we should go back to the original palette or something?

![Color palette still off3](https://user-images.githubusercontent.com/3118014/140397257-09de63ef-413f-4007-8858-07abea831bf1.png)
![Color palette still off4](https://user-images.githubusercontent.com/3118014/140397260-8e1b9c83-bc1f-4337-a164-8919b0b6c67f.png)
![Color palette still off5](https://user-images.githubusercontent.com/3118014/140397261-177e5ed0-2ea4-4b81-b67e-7963cf9e0427.png)
